build-support/ivy/pushdb/org.pantsbuild/ivy-dependency-update-checker/publish.properties

```diff
@@ -0,0 +1,7 @@
+revision.major.org.pantsbuild%ivy-depdendency-update-checker=0
+revision.minor.org.pantsbuild%ivy-depdendency-update-checker=0
+revision.patch.org.pantsbuild%ivy-depdendency-update-checker=1
+revision.snapshot.org.pantsbuild%ivy-depdendency-update-checker=false
```

Comment:
`depdendency`

But also, I don't think that this should have been published at all yet, so this file probably shouldn't be here?